style: apply /style-guide pass to weave (#2739)
wandb/docs: weave/guides/integrations/anthropic.mdx

[![anthropic_trace.png](/weave/guides/integrations/imgs/anthropic_trace.png)](https://wandb.ai/capecape/anthropic_project/weave/calls)

## Wrap with your own ops

> While `weave.init()` alone captures Anthropic SDK calls, wrapping your own functions as ops gives you richer traces that include your application logic around each model call.

Reviewer: ```suggestion
While `weave.init()` alone captures Anthropic SDK calls, decorating your own functions with ops gives you richer traces that include your application logic around each model call.
```

Author: Applied verbatim in eaf82665d.

  </Tab>
</Tabs>

> When you decorate or wrap a function with `weave.op()`, Weave captures the function's code, input, and output. You can use ops to track any function you want, including nested functions.

Reviewer: ```suggestion
When you decorate a function with `weave.op()`, Weave captures the function's code, input, and output. You can use ops to track any function you want, including nested functions.
```

Author: Applied verbatim in eaf82665d.
